deno.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "@netflix/x-element",
   "version": "1.0.0",
-  "exports": "./x-element.js"
+  "exports": {
```

Comment:
this seems wrong?

Reply:
JSR’s documentation is not great here. I’ve added back the other files, which causes JSR to flag us as “slow” and throw warnings. I _think_ that’s ok for now though.

I’m going to follow along on https://github.com/jsr-io/jsr/issues/494 — hopefully things will get better once our JSDoc-style path forward here is accepted.